xml/System.IO/Stream.xml

```diff
@@ -775,7 +775,7 @@
       <Docs>
         <param name="destination">The stream to which the contents of the current stream will be copied.</param>
         <param name="bufferSize">The size of the buffer. This value must be greater than zero. The default size is 81920.</param>
-        <summary>Reads the bytes from the current stream and writes them to another stream, using a specified buffer size.</summary>
+        <summary>Reads the bytes from the current stream, advances the current stream's read position, writes the bytes to the destination stream and advances the destination stream's write position, using a specified buffer size.</summary>
```

Comment:
```suggestion
        <summary>Reads the bytes from the current stream and writes them to another stream, using a specified buffer size. Both streams positions are advanced by the number of bytes copied.</summary>
```

Reply:
Thank you very much! The simplification makes it much easier to understand.